docker/build-tools/Dockerfile

```
@@ -393,11 +393,10 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 # Pinned versions of stuff we pull in
 ENV AWESOMEBOT_VERSION=1.20.0
-# Sometime between 1.14.1 and 1.14.2 to pull in some fixes for deb
-ENV FPM_VERSION=eb5370d16e361db3f1425f8c898bafe7f3c66869
-ENV HTMLPROOFER_VERSION=3.19.0
-ENV NOKOGIRI_VERSION=1.14.3
-ENV LICENSEE_VERSION=9.15.1
+ENV FPM_VERSION=v1.15.1
+ENV HTMLPROOFER_VERSION=3.19.4
+ENV NOKOGIRI_VERSION=1.15.2
```

Our istio.io pages fail proofing on 1.15.x versions.
```suggestion
ENV NOKOGIRI_VERSION=1.14.5
```